activiti-cloud-services-runtime-bundle/activiti-cloud-services-connectors/pom.xml

```diff
@@ -0,0 +1,80 @@
+<!--
+  ~ Copyright 2017 Alfresco, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~       http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+
+  <parent>
+    <groupId>org.activiti.cloud</groupId>
+    <artifactId>activiti-cloud-services-runtime-bundle</artifactId>
+    <version>7-201710-EA-SNAPSHOT</version>
+  </parent>
+
+
+  <artifactId>activiti-cloud-services-connectors</artifactId>
+  <name>Activiti - Services :: Connectors</name>
+  <description>Activiti connectors service</description>
+
+  <dependencies>
+    <dependency>
```

Comment:
@erdemedeiros I'm not happy about this dependency here.. but this is going to be part of the interfaces refactoring of the engine.

Reply:
I understand.